cocotb/share/lib/vpi/VpiCbHdl.cpp

```diff
@@ -458,7 +464,7 @@ int VpiValueCbHdl::cleanup_callback()
     /* This is a recurring callback so just remove when
      * not wanted */
     if (!(vpi_remove_cb(get_handle<vpiHandle>()))) {
-        LOG_CRITICAL("VPI: unbale to remove callback : ABORTING");
+        LOG_CRITICAL("VPI: unbale to remove callback");
```

Comment:
Should this one either:
* end the sim
* be `ERROR` instead?

Reply:
Just missed that, it should be ERROR. Or at least that's what I had it as previously.